src/workerd/api/sql.h

```diff
@@ -33,22 +33,25 @@ class SqlStorage final: public jsg::Object, private SqliteDatabase::Regulator {
   JSG_RESOURCE_TYPE(SqlStorage, CompatibilityFlags::Reader flags) {
     JSG_METHOD(exec);
 
-    // Prepared statement API is experimental-only and deprecated. exec() will automatically
-    // handle caching prepared statements, so apps don't need to worry about it.
     if (flags.getWorkerdExperimental()) {
+      // Prepared statement API is experimental-only and deprecated. exec() will automatically
+      // handle caching prepared statements, so apps don't need to worry about it.
       JSG_METHOD(prepare);
-    }
 
-    // Make sure that the 'ingest' function is still experimental-only if and when
-    // the SQL API becomes publicly available.
-    if (flags.getWorkerdExperimental()) {
+      // 'ingest' functionality is still experimental-only
       JSG_METHOD(ingest);
     }
 
     JSG_READONLY_PROTOTYPE_PROPERTY(databaseSize, getDatabaseSize);
 
     JSG_NESTED_TYPE(Cursor);
     JSG_NESTED_TYPE(Statement);
+
+    JSG_TS_OVERRIDE({
+      prepare: never;
```

Comment:
I don't think we have to explicitly override these to hide them... they will only show up in then first place if someone is generating types with "experimental" enabled, in which case they should actually get those declarations, right?

Reply:
Oh my mistake, I was looking at the wrong output directory.